navigation.js

```diff
@@ -209,22 +192,6 @@ define('navigation',
                 el.getAttribute('rel') === 'external';
     }
 
-    z.body.on('click', 'a', function(e) {
-        var href = this.getAttribute('href');
-        var $elm = $(this);
-        var preserveScrollData = $elm.data('preserveScroll');
-        // Handle both data-preserve-scroll and data-preserve-scroll="true"
-        var preserveScroll = typeof preserveScrollData !== 'undefined' && preserveScrollData !== false;
```

Comment:
Yeah, we're not using it anymore. I can add it back it if it's not too much code and fits in cleanly in case we ever have mid-page tabs.

Yeah, the losing scroll after login is a bug, though I believe that'd have to be handled different since the view reloads entirely.